### ISSUE TITLE
Server zones - Root domain only?

### ISSUE THREAD

**Issue by qcybb** [vozlt/nginx-module-vts#299]
Is there anyway to have this module combine all the stats from the subdomains in to the root domain?

For example, my "Server Zones" list shows :

example.com
www.example.com
*.example.com

I'd like to combine all the stats from those 3 in to one "example.com" column.

Is that possible?

**Comment by u5surf** [comment]
Yes, with a `map` and a filter key. The server zone stays per host; the filter zone is the combined one:

```nginx
map $host $root_domain {
    default                  $host;
    ~^(.*\.)?example\.com$   example.com;
}

server {
    vhost_traffic_status_filter_by_set_key $root_domain root::;
}
```

Requests to `a.example.com`, `www.example.com` and `example.com` then give:

```json
"filterZones": {"root::": {"example.com": {"requestCounter": 3, ...}}}
```

`nginx_vts_filter_*` carries the same in the Prometheus output.

Closing - reopen if you need the merge inside `serverZones` itself rather than beside it.
